pages/publicLibraryFiles/PublicLibraryFileHandler.inc.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * @defgroup controllers_api_file File API controller
+ */
+
+/**
+ * @file controllers/api/file/FileApiHandler.inc.php
```

Comment:
The filename etc. needs updating here (if this stays unchanged).

Reply:
Done